packages/styles/src/index.ts

```diff
@@ -0,0 +1 @@
+export { default as makeCreateStyles } from './private/makeCreateStyles';
```

Comment:
In the future, when we want to make it better, I want to reduce a confusing part:

- `makeCreateStyles` create `HTMLStyleElement[]` and it contains styles from every `*.module.css` in the bundle
- `useStyles` contains class names for only one `*.module.css`, not every in the bundle

Can we pick either one concept?

Reply:
Yea, unfortunately for now this reflects how we bundle things. The design was influenced the actual implementation of the bundling CSS and such